mmv1/products/filestore/api.yaml

```diff
@@ -250,3 +248,7 @@ objects:
           Server-specified ETag for the instance resource to prevent
           simultaneous updates from overwriting each other.
         output: true
+      - !ruby/object:Api::Type::String
+        name: 'kmsKeyName'
```

Comment:
We probably need to add `input: true` here, as this field can not be updated. Adding `input: true` will generate `ForceNew: true` in provider, which means if user want update this field, terraform will delete and recreate the resource.